apps/web/pages/[user]/[type].tsx

```diff
@@ -40,6 +40,7 @@ export default function Type({ slug, user, booking, away, isBrandingHidden, resc
   );
 }
 
+Type.isBookingPage = true;
```

Comment:
New Booker pages were missing this which was causing them to use the same storage key as app and thus app could override booking pages theme if opened side by side.